src/app/reducers/environments.ts

```diff
@@ -17,16 +17,23 @@ export interface State {
 }
 
 export const getInitialEnvironmentState = (): EnvironmentState => {
+  const { initialData } = getAltairConfig();
   return {
     title: 'Environment',
-    variablesJson: '{}'
+    variablesJson: '{}',
+    ...initialData.environments.base
   };
 };
 
+const getInitialSubEnvironmentState = (): EnvironmentState[] => {
+  const { initialData } = getAltairConfig();
+  return initialData.environments.subEnvironments || [];
```

Comment:
Either require the developer to provide the data along with ID (which I wouldn't recommend for many reasons), or map through the subEnvironments and set the id for each using `uuid()`

Reply:
Makes sense. I thought because the ID was nullable on the type that it would auto-populate once initialized. I’ll update.

Reply:
Updated in d0be177